full/Dockerfile

```diff
@@ -23,6 +23,11 @@ RUN yes | unminimize \
 
 ENV LANG=en_US.UTF-8
 
+### Git ###
+RUN add-apt-repository -y ppa:git-core/ppa \
+    && apt-get install -yq git \
+    && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/*
```

Comment:
Note: Only `rm -rf /var/lib/apt/lists/*` is required to clean up after `apt-get`.

Sources:
- https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get
- https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34

Reply:
Good to know, done!